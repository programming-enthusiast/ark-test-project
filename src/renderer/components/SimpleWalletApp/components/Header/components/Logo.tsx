import "../../../index.css";

import React from "react";

import logoImage from "../../../assets/Illustrations/logo.svg";

interface LogoPropType {
	className: string;
}

const Logo = ({ className }: LogoPropType) => (
	<div className={`${className} flex`}>
		<img className={`w-56px h-56px lg:m-auto`} src={logoImage} />
		<h1 className="logo-font inline m-auto ml-10px block lg:hidden xxl:block">
			ARK Wallet
		</h1>
	</div>
);

export default Logo;
