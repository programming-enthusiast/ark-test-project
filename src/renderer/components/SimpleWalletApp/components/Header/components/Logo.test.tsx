import React from "react";
import renderer from "react-test-renderer";

import Logo from "./Logo";

describe("Logo Component", () => {
	let renderedComponent: any;

	beforeAll(() => {
		renderedComponent = renderer.create(
			<Logo className="my-auto ml-40px lg:m-auto " />
		);
	});

	it("should render correctly", () => {
		expect(renderedComponent.toJSON()).toMatchSnapshot();
	});
});
