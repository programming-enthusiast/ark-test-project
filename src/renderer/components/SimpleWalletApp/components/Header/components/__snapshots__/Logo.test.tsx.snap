// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component should render correctly 1`] = `
<div
  className="my-auto ml-40px lg:m-auto  flex"
>
  <img
    className="w-56px h-56px lg:m-auto"
    src="logo.svg"
  />
  <h1
    className="logo-font inline m-auto ml-10px block lg:hidden xxl:block"
  >
    ARK Wallet
  </h1>
</div>
`;
